api/src/pcapi/utils/settings.py

```diff
@@ -30,3 +31,12 @@ def parse_phone_numbers(phone_numbers: str | None) -> list[str]:
     except Exception as exception:  # pylint: disable=broad-except
         logger.exception("Error when parsing phone_numbers variable %s: %s", phone_numbers, exception)
         return []
+
+
+def get_list(key: str, separator: str = ",", type_: type = str) -> list:
```

Comment:
```suggestion
def env_get_list(key: str, separator: str = ",", type_: type = str) -> list:
```
Ou autre nom, pour signifier que l'on va récupérer la variable d'environnement au sein de la fonction